CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# v2.6.0 (2024-09-17)
+
+## Build Changes
+* Validate twoliter upon install ([#147])
+
+## OS Changes
+* Add the ability for driverdog to copy modules ([#1119])
```

Comment:
Add the ability for driverdog to copy modules ([#119]) - Mistyped the PR number.